src/common/cmdparse.cc

```diff
@@ -179,10 +179,20 @@ dump_cmd_to_json(Formatter *f, uint64_t features, const string& cmd)
       desckv["positional"] = "false";
     }
     for (auto [key, value] : desckv) {
-      if (key == "positional" && !HAVE_FEATURE(features, SERVER_QUINCY)) {
-	continue;
+      if (key == "positional") {
+	if (!HAVE_FEATURE(features, SERVER_QUINCY)) {
+	  continue;
+	}
+	if (value == "false" || value == "False") {
+	  f->dump_bool(key, false);
+	}
```

Comment:
```c++
bool is_positional = (value == "false" || value == "False");
f->dump_bool(key, false);
```

i'd suggest make this explicit. instead encoding the default value in our memory/understanding.